backend/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM node:10-alpine
```

Comment:
Is there a reason not to use a node release that is already in maintenance and not active anymore? The frontend build container seems to use node:14 already.

Reply:
Yeah. I intend to update this. The reason why I chose 10 here is because it is the version I use in the Docker Compose setup. So it's the one I tested.